app/src/org/commcare/preferences/CommCareServerPreferences.java

```diff
@@ -96,6 +96,10 @@ public static String getSupportEmailAddress() {
         return getServerProperty(PREFS_SUPPORT_ADDRESS_KEY, CommCareApplication.instance().getString(R.string.support_email_address_default)) ;
     }
 
+    public static String getFormRecordKey() {
```

Comment:
Though I think we can move the remote payload url pref to Developer Prefs as well as it's meant to be a dev feature!